homeassistant/components/telegram_bot/broadcast.py

```diff
@@ -1,7 +1,7 @@
 """Support for Telegram bot to send messages only."""
 import logging
 
-from . import PLATFORM_SCHEMA as TELEGRAM_PLATFORM_SCHEMA, initialize_bot
```

Comment:
we can remove this line and the platform schema being set. They should no longer be treated as platforms.